docs/charts/cumulative_num_comparisons_from_blocking_rules_chart.ipynb

```diff
@@ -157,7 +285,127 @@
+       "</script>"
+      ],
+      "text/plain": [
+       "alt.Chart(...)"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "settings = {\n",
+    "    \"link_type\": \"dedupe_only\",\n",
+    "    \"blocking_rules_to_generate_predictions\": [\n",
+    "        brl.and_(\n",
+    "            brl.exact_match_rule(\"first_name\"),\n",
+    "            brl.exact_match_rule(\"dob\"),\n",
+    "            ),\n",
+    "        brl.exact_match_rule(\"surname\"),\n",
+    "        brl.exact_match_rule(\"email\")\n",
+    "    ]\n",
+    "}\n",
+    "\n",
+    "linker = DuckDBLinker(df, settings)\n",
+    "\n",
+    "linker.cumulative_num_comparisons_from_blocking_rules_chart()"
+   ]
```

Comment:
Do we want to show off the alternative syntax here?
```py
brs = [
        brl.and_(
            brl.exact_match_rule("first_name"),
            brl.exact_match_rule("dob"),
            ),
        brl.exact_match_rule("surname"),
        brl.exact_match_rule("email")
]
linker.cumulative_num_comparisons_from_blocking_rules_chart(brs)
```

Reply:
The `and_` composition can hopefully be removed before the end of the day too.

Reply:
Yea good point, I will add now